include/game_building.h

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
Convert this to a doxygen comment block and use @defgroup to create a module and then use @ingroup game to make it a submodule of the game module. Also use @{ to group the following comments into the module and add @} in the last comment block.